linux-vm-with-cache/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Deploy a simple Linux VM in West US",
```

Comment:
Suggestion: Something along the lines of "Redis Cache with a Linux VM" so that that the app name is used first in the Azure.com index